docs/pages/reference/graphql-api-reference.md

```diff
@@ -19,17 +19,17 @@ Fields that are deprecated are marked with **{warning-solid}**.
 
 <!-- vale off -->
 
-## `Query` type
+## **`Query`** type
```

Comment:
@kaushambisharma This a auto-generated file. Making changes here will not be retained when docs for new graphql schema would be made.

Make changes in the [`default.md.haml`](https://github.com/layer5io/meshery/blob/master/docs/lib/tasks/templates/default.md.haml) and the [`helper.rb`](https://github.com/layer5io/meshery/blob/fa55b66f9ed5d5745206fd2e2c6a2d3fd4d1937c/docs/lib/tasks/helper.rb#L71) file.

After making changes run `make build-graphql-docs` inside doc folder to generate the `graphql-api-reference.md` file.

Reply:
Oh thanks, will work on this.